components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -89,13 +89,115 @@ A [cookbook]({{< relref "cookbooks" >}}) is the fundamental unit of configuratio
 
 An [environment]({{< relref "environments" >}}) can be used to map an organization's real-life workflow to what can be configured and managed when using Chef Infra. This mapping is accomplished by setting attributes and pinning cookbooks at the environment level.
 
+The Chef Infra Server UI lets you:
+
+- Create an environment.
+
+- Search for a specific environment from a list of environments
+
+- List out all the environments at one place.
+
+- View the details of an environment.
+
+- Edit the Environment.
+
+- Delete an environment.
+
+#### Create an Environment
+
+To create a new Environment, select **Create Environment**. Refer to the image below:
+
+{{< figure src="/images/automate/create-environment-button.png" alt="Create Environment Button">}}
+
+Selecting the **Create Environment** button opens a popup menu. The popup menu contains four different sections named _Details_, _Constraints_, _Default Attributes_ and _Override Attributes_. Enter the details in the possible sections and select **Create** to create a new environment.
```

Comment:
Review the Microsoft Style Guide for how to [describing interactions with the UI](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui) and revise the sections where you discuss  the "popup menu" accordingly.